liblangutil/SourceReferenceFormatter.h

```diff
@@ -77,7 +77,7 @@ class SourceReferenceFormatter
 	{
 		return formatExceptionInformation(
 			_error,
-			(_error.type() == Error::Type::Warning) ? "Warning" : "Error",
+			Error::formatErrorCategory(Error::errorCategory(_error.type())),
```

Comment:
Hmm, don't we call this field severity? So perhaps we should consistently use severity and not category everywhere.

Reply:
you mean also in the enum?

Reply:
Yes. I think the enum, the helper functions, docs, etc. should all either use severity or category as the term, otherwise 1-2 years down the line whoever looks at this will be confused.

Reply:
IMO it's fine to keep it as Category in `Error`, cus there the separation feels more like category. Severity is the interpretation that StandardCompiler gives to that. I'm fine with either if consensus.
@cameel wdyt?

Reply:
Good idea. I agree that "severity" fits better than "category".